packages/menu/src/MenuItem/MenuItem.types.ts

```diff
@@ -54,5 +52,5 @@ export interface MenuItemProps {
    * @deprecated - Size no longer has any effect
    */
   // TODO: codemod to remove `size` props from existing implementations
-  size?: Size;
+  size?: 'default' | 'large';
```

Comment:
Why the switch from `Size`?

Reply:
We've deprecated the `size` prop, and I wanted to remove the type def as well. Having the inline types will avoid breaking TS, but let's us remove old files